backend/src/graphql/resolver/CommunityResolver.test.ts

```diff
@@ -1,121 +1,221 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
 /* eslint-disable @typescript-eslint/no-unsafe-call */
 /* eslint-disable @typescript-eslint/no-unsafe-member-access */
 /* eslint-disable @typescript-eslint/unbound-method */
 /* eslint-disable @typescript-eslint/no-explicit-any */
 /* eslint-disable @typescript-eslint/explicit-module-boundary-types */
 
-import { createTestClient } from 'apollo-server-testing'
-import createServer from '@/server/createServer'
-import CONFIG from '@/config'
-
-jest.mock('@/config')
+import { getCommunities } from '@/seeds/graphql/queries'
+import { Community as DbCommunity } from '@entity/Community'
+import { testEnvironment } from '@test/helpers'
 
 let query: any
 
 // to do: We need a setup for the tests that closes the connection
 let con: any
+let testEnv: any
 
 beforeAll(async () => {
-  const server = await createServer({})
-  con = server.con
-  query = createTestClient(server.apollo).query
+  testEnv = await testEnvironment()
+  query = testEnv.query
+  con = testEnv.con
+  await DbCommunity.clear()
```

Comment:
Why clear the DbCommunity beforeAll() and not afterAll().
I would build up the Data in DbCommunity beforeAll and reset/clear DbCommunity afterAll.

Reply:
Or to be sure doing both

Reply:
all test cases depend on each other, so only a beforeAll is important to have a clear state of database.
not deleting afterAll was to check the correct values in DB after the tests.